chainerx_cc/chainerx/python/array.cc

```diff
@@ -468,6 +467,9 @@ void InitChainerxArray(pybind11::module& m) {
           py::is_operator());
     c.def("__rshift__", [](const ArrayBodyPtr& self, Scalar rhs) { return MoveArrayBody(Array{self} >> rhs); }, py::is_operator());
     c.def("__rrshift__", [](const ArrayBodyPtr& self, Scalar lhs) { return MoveArrayBody(lhs >> Array{self}); }, py::is_operator());
+}
+
+void InitChainerxCalculation(py::class_<ArrayBody, ArrayBodyPtr> c) {
```

Comment:
Rename this function and call it from `InitChainerxArray`.
```suggestion
void InitChainerxArrayCalculation(py::class_<ArrayBody, ArrayBodyPtr> c) {
```

Reply:
Made all the changes. Tests pass successfully now. Thanks for the review.